auth.php

```diff
@@ -597,6 +602,30 @@ public function saml_login() {
         return;
     }
 
+    /**
+     * Testing user's groups attribute and allow access decided on preferences.
+     *
+     */
+    public function is_access_allowed_for_member($attributes) {
+
+        if (empty($this->config->groupattr)) {
+            $this->log(__FUNCTION__ . " group attribute not found.");
```

Comment:
should there really be a logging event here? - or was this left over from initial debuging?